### PR TITLE
Backend: Setup testing infrastructure for database, CI/CD (Pytest, Gitlab mirror)

### PR DESCRIPTION
**Pull request by pojntfx**
Closes https://github.com/alphahorizonio/consumat.io/issues/139, closes https://github.com/alphahorizonio/consumat.io/issues/153

### CODE REVIEW

File: tests/utils/setup_app.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    # Drop database
+    db.reflect()
+    db.drop_all()
+
+    # Start app again for testing
+    app.configure()
+    app.app.app_context().push()
+
+    # Return any parts of the app to used during testing
```
Comment (JakWai01) [inline tests/utils/setup_app.py:37]: It's a bit picky but could you maybe correct the grammar in this line? :smile:

**Review by JakWai01 — CHANGES_REQUESTED** [review]
Please look at the requested change. Also, is there any case where the setup_app function could drop the in-production database during CI/CD?

File: tests/utils/setup_app.py (hunk 1 above)
Comment (pojntfx) [reply tests/utils/setup_app.py:37]: Of course, on moment xD

Comment (pojntfx) [reply tests/utils/setup_app.py]: @JakWai01 Done

**Review by JakWai01 — APPROVED** [review]
LGMT! I'll merge ASAP.

**Pull request merged**